Formula/h/hckr.rb

```diff
@@ -0,0 +1,23 @@
+class Hckr < Formula
+  include Language::Python::Virtualenv
+
+  desc "hckr - Awesome CLI for developers"
+  homepage "https://pypi.org/project/hckr/"
+  url "https://files.pythonhosted.org/packages/source/h/hckr/hckr-0.2.4.tar.gz"
+  sha256 "21d40e1275e05cf43a6cc0f6f72b15c378c1011fcc4f8577d3658c3a2221fbb9"
+  license "MIT"
+
+  depends_on "python@3.8"
```

Comment:
why not py3.12?

Reply:
The minimum python  version required for this is 3.8.
I am new here, suggest if 3.12 will be better.

Reply:
Updated to 3.12